### DISCUSSION TITLE
Note names - re-naming

### DISCUSSION THREAD

**Discussion by dukester** [xwmx/nb#347] [Q&A] [answered]
Just installed `nb'. Got one test note, but I don't like timestamp names- they give zero clue to the content. No joy renaming the note with: `nb move home: 1 --to-title`. I messed up, but don't see where.

Also, is there a way to NOT use timestamps as the default filename?
TIA for the swell app and any help.

**Comment by boldandbusted** [comment]
So, I don't know what's going on under the hood, but `nb` seems to take the name of the note from the first line (and it is Markdown-aware). If you change the first line of the note after `nb edit <note number>`, and run `nb list` or `nb` with no arguments (which defaults to `nb list`), you should see the note now is named for what you put in the first line of the note. Hope this helps.

**Comment by dukester** [comment]
On Thu, 16 Jan 2025 13:12:00 -0800
Jesse Adelman ***@***.***> wrote:

> So, I don't know what's going on under the hood, but `nb` seems
> to take the name of the note from the first line (and it is
> Markdown-aware). If you change the first line of the note after
> `nb edit <note number>`, and run `nb list` or `nb` with no
> arguments (which defaults to `nb list`), you should see the note
> now is named for what you put in the first line of the note. Hope
> this helps.

It does! Thanks a bunch ....
--
Duke

**Reply by boldandbusted** [reply]
I am playing around with nb today, and tried to see if I can make it work with more than one person. Seems like the answer is generally yes, but it showed me that having filenames that are unique is really important when working with a team. So, the timestamped filenames (with seconds!) makes it less likely there will be a filename collision. Just keep that in mind if you work with others. And, yes, that means if you think there's even a small chance of the notebook becoming a shared resource then your naming convention may haunt you if you aren't careful... Cheers.

**Reply by boldandbusted** [reply, marked as answer]
Also, please mark this answer as "Answered" if it does answer your initial query. :)

**Reply by dukester** [reply]
Thx ...
